RxCocoa/iOS/UIStepper+Rx.swift

```diff
@@ -26,6 +26,13 @@ extension Reactive where Base: UIStepper {
             }
         )
     }
+
+    /// Reactive wrapper for `stepValue` property.
+    public var stepValue: UIBindingObserver<Base, Double> {
+        return UIBindingObserver(UIElement: self.base) { stepper, value in
```

Comment:
Don't we need to observe this value? I think we have to make this `ControlProperty` instead of `UIBindingObserver`.